### PR TITLE
Add operator to the proposal

### PR DESCRIPTION
**Pull request by ThunderDeliverer**
The operator parameter was added to isTransferable function. The isTransferable method that only accepted the tokenId was removed. Updated specification was propagated throughout the proposal and the assets.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +67,16 @@ interface IERC6454 /* is IERC165 */ {
 }
 ```
 
+In order to determine whether a token is transferable or not in general, the function SHOULD return the appropriate boolean value when passing the `0x0000000000000000000000000000000000000000` address as the `to` and `from` parameter.
+
+The general transferability of a token should not be affected by the ability to mint the token (value of `from` parameter is `0x0000000000000000000000000000000000000000`) and the ability to burn the token (value of `to` parameter is `0x0000000000000000000000000000000000000000`).
+
+If the general transferability of token is `false`, any kind of transfer of the token, save minting and burning, MUST revert execution.
+
+In order to determine whether a token is mintable, the exception SHOULD be made to allow the `tokenId` parameter for a token that does not exist. Additionally the `to` parameter SHOULD be `0x0000000000000000000000000000000000000000` and the `from` parameter SHOULD NOT be `0x0000000000000000000000000000000000000000`.
```
Comment (steven2308) [inline README.md:76]: I feel this is the opposite. To determine if it is mintable, from must be 0 and to != 0

File: README.md [hunk 2]
```diff
@@ -75,6 +67,16 @@ interface IERC6454 /* is IERC165 */ {
...
+
+In order to determine whether a token is burnable, the `to` parameter SHOULD NOT be `0x0000000000000000000000000000000000000000` and the `from` parameter SHOULD be `0x0000000000000000000000000000000000000000`.
```
Comment (steven2308) [inline README.md:78]: Similarly here, seems the other way around.

File: README.md [hunk 3]
```diff
@@ -85,11 +87,11 @@ Designing the proposal, we considered the following questions:
...
+5. **Should we include the most straightforward method possible that only accepts a `tokenId` parameter?**\
+   The initial version of the proposal contained a method that only accepted a `tokenId` parameter. This method would return a boolean value indicating whether the token is transferable. However, the fact that the token can be non-transferable for different reasons was discussed throughout the discussion. This is why the method was changed to accept additional parameters, allowing for a more flexible implementation. Additionally, we kept the original method’s functionality by specifying the methodology on how to achieve the same result (by passing the `0x0000000000000000000000000000000000000000` address as the `to` and `from` parameters).
```
Comment (steven2308) [inline README.md:94]: "was discussed throughout the discussion". Could you rephrase it so it is not repetitive?

File: README.md (hunk 1 above)
Comment (ThunderDeliverer) [reply README.md:76]: Right 🤦

**Review by steven2308 — APPROVED** [review]
